css/buzznom.css

```diff
@@ -0,0 +1,170 @@
+
+body {
+	background-color:#fff;
+	background-image:url(../img/beeLogo.png);
+	background-repeat:no-repeat;
+	background-position: 30px 30px;
+	width:100%;
+	margin:0;
+	border:0;
+}
+#honeyComb {
+	width: 100%;
+	text-align:center;
+}
+.row {
+	width:900px;
+	height:200px;
+	display:inline-block;
+	margin: 10px 10px 10px 10px;
+}
+.row:nth-child(1) {
+margin-top:150px;
+margin-left:220px;
+}
+.row:nth-child(2) {
+margin-top:-40px;
+}
+.row:nth-child(3) {
+margin-top:-40px;
+margin-left:220px;
+}
+#show-more {
+	background-color:#d29928;
+	cursor:pointer;
+}
+.flip-side {
+	display:none;
+	margin-top:-10px;
+}
+.display-flip-side .flip-side{
+	display:block;
+}
+.display-flip-side {
+   background-image:url(../img/honey.png) !important;
+}
+#zip {
+	position:absolute;
+	width:100%;
+}
+#zip input {
+	margin:15px;
+	float:right;
+	text-align: center;
+}
+
+
+
+
+/* Hexagon CSS Author: Kevin Liew */
```

Comment:
Have a URL for this? Make sure to include it when you use code from elsewhere, so that it's easier for other people to look up.